Framework/PythonInterface/mantid/kernel/src/Exports/V3D.cpp

```diff
@@ -60,13 +62,17 @@ void export_V3D() {
            "Calculates the length of the vector")
       .def("norm2", &V3D::norm2, arg("self"),
            "Calculates the squared length of the vector")
-      .def(self + self)
-      .def(self += self)
-      .def(self - self)
-      .def(self -= self)
+      .def("__add__", &V3D::operator+, (arg("left"), arg("right")))
+      .def("__iadd__", &V3D::operator+=, return_self<>(),
+           (arg("self"), arg("other")))
+      .def("__sub__", &V3D::operator-, (arg("left"), arg("right")))
+      .def("__isub__", &V3D::operator-=, return_self<>(),
+           (arg("self"), arg("other")))
       .def(self * self)
```

Comment:
This looks like overloaded operators. So there is more work than just simple substitution. See for example https://cs.brown.edu/~jwicks/boost/libs/python/doc/tutorial/doc/html/python/functions.html, the section about overloading